.drone.yml

```diff
@@ -9,12 +9,14 @@ platform:
 steps:
 - name: build
   image: rancher/dapper:v0.5.0
-  secrets: [ gcloud_auth, unprivileged_github_token ]
+  secrets: [ aws_secret_access_key, aws_access_key_id, unprivileged_github_token ]
```

Comment:
Shouldn’t this be the name of the drone secret? 
`aws_secret_access_key => AWS_SECRET_ACCESS_KEY-artifact-uploader`
just like it's been done for `unprivileged_github_token`

Reply:
Done.